docs/reference/execution-providers/DNNL-ExecutionProvider.md

```diff
@@ -12,21 +12,21 @@ Intel® Math Kernel Library for Deep Neural Networks (Intel® DNNL) is an open-s
 
 Intel and Microsoft have developed DNNL Execution Provider (EP) for ONNX Runtime to accelerate performance of ONNX Runtime using Intel® Math Kernel Library for Deep Neural Networks (Intel® DNNL) optimized primitives.
 
-
 ## Contents
 {: .no_toc }
```

Comment:
Not sure if {: .no_toc} this is supposed to be here, the master branch doesn't currently reflect this. See https://github.com/microsoft/onnxruntime/blob/master/docs/execution_providers/DNNL-ExecutionProvider.md.

DML, Nuphar, and NNAPI, OpenVINO, RKNPU, MiGraphX

Reply:
> Not sure if {: .no_toc} this is supposed to be here, the master branch doesn't currently reflect this. See https://github.com/microsoft/onnxruntime/blob/master/docs/execution_providers/DNNL-ExecutionProvider.md.
> 
> DML, Nuphar, and NNAPI, OpenVINO, RKNPU, MiGraphX

We resolved this offline. The {: :no_toc} prevents the Contents heading from being added to the TOC in the published docs.